admin_guide/cluster_management.rst

```diff
@@ -145,3 +145,24 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
     102240,
     'source_host', source_port,
     'dest_host', dest_port);
+
+Database Bloat
+##############
+
+It’s not just user queries which scale in a distributed database, vacuuming does too. In PostgreSQL big busy tables have great potential to bloat, both from lower sensitivity to PostgreSQL's vacuum scale factor, and generally because of the extent of their row churn. Splitting a table into distributed shards means both that individual shards are smaller tables and that auto-vacuum workers can parallelize over different parts of the table on different machines. Ordinarily auto-vacuum can only run one worker per table.
```

Comment:
Sure, I can add an intro about bloat.

The vacuum scale factor I mentioned is a database config parameter that affects when autovacuum decides to start working on a table: it starts for every n rows written where n = autovacuum_vacuum_scale_factor * [reltuples in pg_class for table] + autovacuum_vacuum_threshold. So autovacuum won't begin until many rows get updated on a big table unless the scale factor is decreased.